src/main/java/net/zhuoweizhang/raspberryjuice/RemoteSession.java

```diff
@@ -648,6 +846,15 @@ private Location parseLocation(World world, int x, int y, int z, int originX, in
 		return new Location(world, originX + x, originY + y, originZ + z);
 	}
 
+	private int getDistance(Entity ent1, Entity ent2) {
+		if (ent1 == null || ent2 == null)
+			return -1;
+		double dx = ent2.getLocation().getX() - ent1.getLocation().getX();
+		double dy = ent2.getLocation().getY() - ent1.getLocation().getY();
+		double dz = ent2.getLocation().getZ() - ent1.getLocation().getZ();
+		return (int)Math.sqrt(dx*dx + dy*dy + dz*dz);
```

Comment:
Why cast to an int? why not have the precise number?

Reply:
Just to have distance in whole blocks. But It can return double. It can be always casted to int if required. I agree with you.